src/types/tmdb.ts

```diff
@@ -0,0 +1,85 @@
+export interface MovieData {
+    page: number;
+    results: MovieResultData[];
+    total_pages: number;
+    total_results: number;
+}
+
+export interface MovieResultData {
+    adult: boolean;
+    backdrop_path: string;
+    genre_ids: number[];
+    id: number;
+    original_language: string;
+    original_title: string;
+    overview: string;
+    popularity: number;
+    poster_path: string;
+    release_date: string;
+    title: string;
+    video: boolean;
+    vote_average: number;
+    vote_count: number;
+}
+
+export interface MovieDetailsData extends MovieResultData {
+    belongs_to_collection: {
+        backdrop_path: string,
```

Comment:
So this 100% works. But the TS Handbook uses semi colons for line endings within interfaces. I think we should at least stay standard across the app. Not sure if there is an actual reason to use semi colons over commas. https://www.typescriptlang.org/docs/handbook/interfaces.html